### PR TITLE
Feature/gitbook start

### PR DESCRIPTION
**Pull request by wimpers**

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # VolumeDriver Config file
-The VolumeDriver gets configured per vPool. To change the config file, edit the json. Some of the values can be dynamically changed (dynamic reconfigurable), other values require a restart. To make sure the static values get applied execute ** service ovs-volumedriver-vpool_name restart**
+The VolumeDriver gets configured per vPool. To change the config update the values in ETCD `/ovs/vpools/<vpool guid>`. Some of the values can be dynamically changed (dynamic reconfigurable), other values require a restart. To make sure the static values get applied execute ** service ovs-volumedriver-vpool_name restart**
```
Comment (redlicha) [inline docs/config.md]: * Better(?): "The VolumeDriver is configured per vPool."
* s/dynamic reconfigurable/dynamically reconfigurable/
* Better(?): "To make sure the values that cannot be reconfigured dynamically are applied [...]"

**Comment by redlicha** [comment]
Besides the one comment the non-edge / libovsvolumedriver look good to me.
@cnanakos, please review the latter.

**Comment by cnanakos** [comment]
@wimpers  You need to rebase this feature branch, it has some conflicts.

**Comment by wimpers** [comment]
@redlicha can you please review so I can merge this one now that there are no conflicts.

**Comment by redlicha** [comment]
The new changes look ok to me.

**Pull request merged**